content/blog/2022-12-06-keda-2.9.0-release.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Announcing KEDA v2.9 🎉"
+date = 2022-12-08
+author = "Jeff Hollan (KEDA), Tom Kerkhove (KEDA)"
+aliases = [
+"/blog/2.9.0-release"
+]
++++
+
+We recently completed our most recent release: 2.9.0 🎉!
+
+Here are some highlights:
+
+- A pre-made Grafana dashboard to visualize metrics exposed by the KEDA metrics adapter ([docs](https://keda.sh/docs/2.9/operate/prometheus/))
+- New Loki Scaler ([docs](https://keda.sh/docs/scalers/loki/))
+- All metrics exposed to prometheus are now consolidated in KEDA Operator
+- Azure Key Vault now suppoerts pod identities for authentication
+
+Potential breaking changes include:
+- Upgrading the API version of managed HPA objects from `autoscaling/v2beta2` to `autoscaling/v2`
+
+There are no deprecations to announce as part of this release.
```

Comment:
There is one which we are adding and should list all other active ones:
https://github.com/kedacore/keda/pull/3974

```suggestion
We are announcing the deprecation of our Prometheus metrics on the KEDA metrics adapter in favor of the new metrics on our KEDA operator. Learn more in [this issue](https://github.com/kedacore/keda/issues/3972) and [join the conversation](https://github.com/kedacore/keda/discussions/3973).
```